server/src/main/java/io/crate/metadata/doc/DocSchemaInfo.java

```diff
@@ -270,10 +275,12 @@ public void update(ClusterChangedEvent event) {
 
             IndexMetadata newIndexMetadata = newMetadata.index(indexName);
             if (newIndexMetadata == null) {
+                LOGGER.debug("Invalidating DocTableInfo for deleted table [{}] [{}]", tableName, newMetadata.indices().keys());
```

Comment:
Avoid `keys()` computation if debug isn't enabled.
```suggestion
                if (LOGGER.isDebugEnabled()) {
                    LOGGER.debug("Invalidating DocTableInfo for deleted table [{}] [{}]", tableName, newMetadata.indices().keys());
                }
```